src/services/TestDataService.ts

```diff
@@ -1234,6 +1235,54 @@ export class TestDataService {
         return salesChannel;
     }
 
+    /**
+     * Creates a new domain for a sales channel.
+     * 
+     * @param overrides - Specific data overrides that will be applied to the sales channel domain data struct.
+     */
+    async createSalesChannelDomain(overrides: Partial<SalesChannelDomain> = {}): Promise<SalesChannelDomain>  {
```

Comment:
could you please put that method to the area where all other create methods are located in the TestDataService?